lib/newman_conway.py

```diff
@@ -1,10 +1,32 @@
 
-
+import array
 # Time complexity: ?
 # Space Complexity: ?
 def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: o(n)
+        Space Complexity: o(n)
     """
-    pass
+    # to store values 
+    if num < 1: 
+        raise ValueError ("Invalid Num")
+    if num == 1: 
+        return "1"
+
+    f = [0, 1, 1]
+    output = "1"
+
+    count = 3 
+    # To store values of sequence in array
+
+    while count <= num: 
+        f.append(f[f[count - 1]]+ f[count - f [ count -1]])
+        count += 1 
+    number = [str(item) for item in f]
+    
+    return " ".join(number[1:])
```

Comment:
✨ Nice use of a list comprehension to convert the numeric values to strings.

Another approach would be to use the `map` function:
```py
    return " ".join(map(str, number[1:]))
```